packaging/ansible-runner-service-project/project/roles/ovirt-host-enroll-certificate/tasks/main.yml

```diff
@@ -5,3 +5,10 @@
     - vmconsole
     - ovn
     - vnc
+
+- name: Update certs permissions
+  file:
+    path: '{{ ovirt_pki_dir }}'
+    mode: u=rwx,g=r,o=r
+    recurse: yes
```

Comment:
This should be more fine-grained.  The key files there and maybe some directories shouldn't be o=r and in some cases also g=r.